packages/client/src/views/GrantDetails.vue

```diff
@@ -58,19 +58,20 @@
                 </b-button>
                 <b-button
                   class="w-50 flex-shrink-1"
-                  variant="outline-primary"
+                  :variant="copyUrlSuccessTimeout === null ? 'outline-primary' : 'outline-success'"
                   @click="copyUrl"
                 >
-                  <b-icon icon="files" aria-hidden="true" class="mr-2" />
-                  Copy Link
+                  <b-icon :icon="copyUrlSuccessTimeout === null ? 'files' : 'check2'" aria-hidden="true" class="mr-2" />
+                  <span v-if="copyUrlSuccessTimeout === null">Copy Link</span>
+                  <span v-else>Link Copied</span>
```

Comment:
The Figma design has this text as "Successfully copied" - everything else looks good here

Reply:
Yeah, I made a game time decision once I had this implemented and could see it in action to go with a wording change. To my eye, the "Successfully Copied" text made the transition more jarring, crowded, and hard to read. 

I'm used to making small tweaks to designs like that as I go, but if that's not the norm here, I'm happy to adjust! And I probably should have called it out in the description. 

Anyway, easy change, I'll implement the language from the design. :)